src/drivers/sqlite/pool.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const EventEmitter = require('events');
+const Connection = require('./connection');
+
+class Pool extends EventEmitter {
+  constructor(opts) {
+    super(opts);
+    this.options = {
+      connectionLimit: 10,
+      ...opts,
+      client: opts.client || 'sqlite3',
+    };
+    this.client = require(this.options.client);
+    this.connections = [];
+    this.queue = [];
+  }
+
+  async getConnection() {
+    const { connections, queue, client, options } = this;
+    for (const connection of connections) {
+      if (connection.idle) {
+        connection.idle = false;
+        this.emit('acquire', connection);
+        return connection;
+      }
+    }
+    if (connections.length < options.connectionLimit) {
+      const connection = new Connection({ ...options, client, pool: this });
+      connections.push(connection);
+      this.emit('connection', connection);
```

Comment:
`connection` is not a verb, should we use `connect` or `alloc`?

Reply:
The name was stolen from [mysqljs/mysql](https://github.com/mysqljs/mysql#connection), personally I'd agree with verb over noun, but let's try keep the event names consistent across drivers for now.